hack/tools/test_module.sh

```diff
@@ -0,0 +1,201 @@
+    --namespace cert-manager \
+    --version v1.2.0 \
+    --create-namespace \
+    --set installCRDs=true \
+    --wait --timeout 400s
+
+
+
+bin/helm install vault fybrik-charts/vault --create-namespace -n fybrik-system \
+        --set "vault.injector.enabled=false" \
+        --set "vault.server.dev.enabled=true" \
+        --values https://raw.githubusercontent.com/fybrik/fybrik/v0.5.3/charts/vault/env/dev/vault-single-cluster-values.yaml
+    bin/kubectl wait --for=condition=ready --all pod -n fybrik-system --timeout=400s
+
+bin/helm install fybrik-crd fybrik-charts/fybrik-crd -n fybrik-system --version v$fybrikVersion --wait
+bin/helm install fybrik fybrik-charts/fybrik -n fybrik-system --version v$fybrikVersion --wait
+
+
+# cd /data/checkagain14/data-movement-operator/
+if [ $fybrikVersion == "0.6.0" ]
+then
+    git clone https://github.com/fybrik/data-movement-operator.git
+    cd data-movement-operator/
+    git checkout releases/0.6.0
+    bin/helm install data-movement-operator charts/data-movement-operator -n fybrik-system --wait
+    cd ..
+    rm -rf data-movement-operator
+fi
+#cd /data/fybrik-release-0.5.0/fybrik
+#helm install fybrik-crd charts/fybrik-crd -n fybrik-system --wait
+#helm install fybrik charts/fybrik --set global.tag=0.5.3 --set global.imagePullPolicy=Always -n fybrik-system --wait
+
+
+
+# apply modules
+#kubectl apply -f https://github.com/fybrik/arrow-flight-module/releases/latest/download/module.yaml -n fybrik-system
+# kubectl apply -f ${WORKING_DIR}/arrow_$moduleVersion_noactions.yaml -n fybrik-system
+
+bin/kubectl apply -f https://github.com/fybrik/arrow-flight-module/releases/download/v$moduleVersion/module.yaml -n fybrik-system
+#kubectl apply -f ${WORKING_DIR}/flight.yaml -n fybrik-system
+if [ $module == "stream" ]
+then
+    bin/kubectl apply -f https://github.com/fybrik/data-movement-operator/releases/download/v$moduleVersion/implicit-copy-stream-module.yaml -n fybrik-system
+else
+    bin/kubectl apply -f https://github.com/fybrik/data-movement-operator/releases/download/v$moduleVersion/implicit-copy-batch-module.yaml -n fybrik-system
+fi
+# kubectl apply -f /data/checkagain14/data-movement-operator/modules/implicit-copy-batch-module.yaml -n fybrik-system
+
+
+# kubectl apply -f ${WORKING_DIR}/arrow_$moduleVersion_noactions.yaml -n fybrik-system
+#kubectl apply -f ${WORKING_DIR}/flight.yaml -n fybrik-system
+# kubectl apply -f https://github.com/fybrik/data-movement-operator/releases/download/v$moduleVersion/implicit-copy-batch-module.yaml -n fybrik-system
+# kubectl apply -f ${WORKING_DIR}/arrow_0.6.0_noactions.yaml -n fybrik-system
+# kubectl apply -f /data/checkagain14/data-movement-operator/modules/implicit-copy-batch-module.yaml -n fybrik-system
+
+
+#datashim
+bin/kubectl apply -f https://raw.githubusercontent.com/datashim-io/datashim/master/release-tools/manifests/dlf.yaml
+bin/kubectl wait --for=condition=ready pods -l app.kubernetes.io/name=dlf -n dlf --timeout=500s
```

Comment:
for dlf please use the dlf.yaml from this commit: https://github.com/revit13/the-mesh-for-data/commit/9fa9af6b0bb0e8a5a86a8d2d38575db4874d0615